tests/improc/test_zogy.py

```diff
@@ -269,6 +270,7 @@ def test_subtraction_snr_histograms(headless_plots):
     plt.savefig(os.path.join(plot_path, "zogy_snr_histograms.png"))
 
 
+@pytest.mark.skip( reason="This test frequently fails even with the flaky.  Can we use a random seed?" )
```

Comment:
I don't think we should set a constant seed, because this test should pass statistically. If we limit to one random seed it may blind us to the fact that most of the time it doesn't work. 

I am surprised this fails so often. I haven't seen any failures since I added the flaky mark. Maybe instead just put a higher `max_runs` like 5?

I'm generally worried, that I put pretty large margins on this test and it should pass most of the time, but it doesn't. 
I think skipping it altogether will lead us to forget about it completely, so I prefer to increase the `max_run` instead.

Reply:
I'm going to leave it as skipped for here; probably we should create an issue to look into this.